package.json

```
@@ -10,6 +10,7 @@
   },
   "devDependencies": {
     "@automattic/color-studio": "^2.5.0",
+    "@babel/eslint-parser": "^7.15.4",
```

Let's pin all the versions of packages we just added, i.e. remove the `^` at the start.
For this specific package let's make this change:

```suggestion
    "@babel/eslint-parser": "7.15.4",
```

And let's apply that to all the other updated/new packages.

---

Sounds good, done in 05bc76cb